### ISSUE TITLE
Question about 02-match-extract-strings.md

### ISSUE THREAD

**Issue by doujouDC** [LibraryCarpentry/lc-data-intro#220]
### How could the content be improved?

Hi, I am teaching this lesson for the first time next month and i want to make sure I am not missing someone in this section: Taking spaces into consideration
Type community . You get three matches. Why not six?

The solution says "The string 'community-led' matches the first search, but drops out of this result because the space does not match the character -." but we are asked to type in "community", not "community-led". Am I missing something?

Actually, I think I figured this, it was not clear that you needed to type community with a space. I will mention this when teaching the lesson

### Which part of the content does your suggestion apply to?

https://github.com/LibraryCarpentry/lc-data-intro/blob/main/episodes/02-match-extract-strings.md

**Comment by doujouDC** [comment]
Hey, I think I understand the issue that I had with this section a little better. However, I did notice that the link in the following sections no longer works: "Find the latest list [here](https://stats.research.icann.org/dns/tld_report/). What expression would capture this? Hint: the . is also a metacharacter, so you will have to use the escape \ to express a literal period. Note: for the string after the period, we did not try to match a - character, since those rarely appear in the characters after the period at the end of an email address." This is on the following page: https://librarycarpentry.org/lc-data-intro/02-match-extract-strings.html#exercise-finding-email-addresses-using-regex101.com
